_posts/2016-05-24-journal-of-a-junior-bem.md

```diff
@@ -0,0 +1,79 @@
+
+The reason for double underscores and hyphens is because if your block or element has a hyphen in its name then you don’t want to confuse the name with a modifier. Like if you had a block called nav-link, it’d be BEM’d up like:
+
+<pre><code class="scss">
+.nav-link {} /* Block */
+.nav-link__field {}
+.nav-bar--active {}
+
+</code></pre>
+<br/>
+
+###What are the benefits of using BEM compared to my previous approach?
+
+My previous approach to stylesheets was very..I’m going to go with amature. I enjoyed writing them but I had no structure to follow. Everything was thrown into one stylesheet and there was a lot of specifics and nesting which I have now learned isn’t a fantastic idea. If I was to change an ID, then I could potentially ruin the style, if I was to change the class of a parent element I could throw off the whole section. There was tons of nesting and so a lot of repeated styles. This lead to a very confusing and daunting stylesheet (but I was proud of it anyways because “Look at all this cool stuff I wrote!”).
+
+The beautiful thing with BEM is that it eliminates a lot of those problems - even though it’s kinda ugly on the outside. It doesn’t need nesting, because a BEM CSS class is independent and doesn’t need tags or IDs to show who it is! It depends on its unique class and its modifiers. 
+
+Another nice quality about BEM is that it’s very informative, you can tell what the style is just by looking at the class. Let’s go with the analogy of a cat:
+
+<pre><code class="scss">
+.cat {}
+.cat__ear{}
+.cat--fat{}
+.cat--fat__ear{}
+.cat__ear--left{}
+
+</code></pre>
+<br/>
+
+
+The block is a cat. The element is its ear. The modifier is that it’s a fat cat (the best kind of cat). You can tell all this just by glancing. Previously I would have written this like:
+
+<pre><code class="scss">
+.cat{}
+.fat{}
+.ear{}
+.left-ear{}
+.right-ear{}
+
+</code></pre>
+<br/>
+
+Reading that and knowing it’s a cat makes sense, but what if you didn’t know? What would the .ear be for - could be a mouse? You can get .fat anything, could it be a fat person? By using BEM you are giving whoever is looking at your markup a good idea of what’s going on and all the different elements relate together but aren't depending on each other!
+
+###And so I conclude..
+
+I really enjoyed learning about BEM (I have this video by Kevin Mack to thank for that), and I can now see why Space Between practice it. It’s informative and I’m really looking forward to applying it to future projects as I can see how much it’s going to benefit me  - it will make me think a lot more about what i’m going to name my classes and help me interpret designs better if i’m thinking of it as blocks and elements. The BEM classnames will help make the markup a lot more descriptive and meaningful especially for my coworkers as we’ll all be following the same syntax.
```

Comment:
Can you link to the video here, please? Also, link to his Twitter page or something :)